### PR TITLE
Added Wacom Graphire2 tablet drivers (latest compatible driver)

### PR DESCRIPTION
**Pull request by ignlg**

### CODE REVIEW

File: Casks/wacom-graphire2-tablet.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+cask :v1 => 'wacom-graphire2-tablet' do
+  version '6.1.7-4'
+  sha256 "593cdd4c51bee7714aecbef77d6e3809dd80a8393893ee1937f9c15c567bb4b4"
```
Comment (vitorgalvao) [inline Casks/wacom-graphire2-tablet.rb]: Double quotes are reserved for cases where string interpolation is needed. Please change these to single quotes.

**Comment by adityadalal924** [comment]
@vitorgalvao Any problems here? 

Wasn't sure why this was left unmerged (except for the fact that commits need to be squashed, of course)

**Comment by vitorgalvao** [comment]
@adityadalal924 No problem anymore (didn’t notice the previous issue had been fixed). Good to (squash and) merge.

**Comment by adityadalal924** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 3521d29bf2a515d638f76b49494384f70f77dc3e. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**